Origin: b129ab51e701af22dd5a046864f0d8a65252c669
Applied-Upstream: b129ab51e701af22dd5a046864f0d8a65252c669
Description: Avoid deprecated API usage in test runner.

Index: ufw-0.36.2/tests/unit/support.py
===================================================================
--- ufw-0.36.2.orig/tests/unit/support.py
+++ ufw-0.36.2/tests/unit/support.py
@@ -111,7 +111,7 @@ def run_unittest(*classes):
 
     suite = unittest.TestSuite()
     for cls in classes:
-        suite.addTest(unittest.makeSuite(cls))
+        suite.addTest(unittest.TestLoader().loadTestsFromTestCase(cls))
 
     runner = unittest.TextTestRunner(sys.stdout, verbosity=2)
     result = runner.run(suite)
